### DISCUSSION TITLE
[2021.09.08] 이름에 el이 들어가는 동물 찾기

### DISCUSSION THREAD

**Discussion by chaeyeon4u** [pyunji/Hyundai-ITnE-Team_QuadCore#120] [SQL]
[이름에 el이 들어가는 동물 찾기](https://programmers.co.kr/learn/courses/30/lessons/59047)

**Comment by hyk339** [comment]
[이름에 el이 들어가는 동물 찾기](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/sql/PR_59047.sql)

**Comment by pyunji** [comment]
[solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/sql/PR_59413.sql)

**Comment by chaeyeon4u** [comment]
[이름에 el이 들어가는 동물 찾기](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/SQL/PR_59047.sql)

**Comment by leechang4** [comment]
[이름에 el이 들어가는 동물 찾기](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/SQL/PR_59047.sql)
